eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -117,3 +117,103 @@ function Process-ReviewStatusCode($statusCode, $packageName, $apiApprovalStatus,
   $packageNameStatus.IsApproved = $packageNameApproved
   $packageNameStatus.Details = $packageNameApprovalDetails
 }
+
+function Set-ApiViewCommentForRelatedIssues {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$SourceCommish,
+    [Parameter(Mandatory = $true)]
+    [string]$HeadCommitish,
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+  )
+  . ${PSScriptRoot}\..\common.ps1
+  $issuesForCommit = $null
+  try {
+    $issuesForCommit = Search-GitHubIssues -CommitHash $HeadCommitish
+    if ($issuesForCommit.items.Count -eq 0) {
+      LogError "No issues found for commit: $HeadCommitish"
+      exit 1
+    }
+  } catch {
+    LogError "No issues found for commit: $HeadCommitish"
+    exit 1
+  }
+  $issuesForCommit.items | ForEach-Object {
+    $urlParts = $_.url -split "/"
+    Set-ApiViewCommentForPR -RepoOwner $urlParts[4] -RepoName $urlParts[5] -PrNumber $urlParts[7] -SourceCommish $SourceCommish -AuthToken $AuthToken
+  }
+}
+
+function Set-ApiViewCommentForPR {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$RepoOwner,
+    [Parameter(Mandatory = $true)]
+    [string]$RepoName,
+    [Parameter(Mandatory = $true)]
+    [string]$PrNumber,
+    [Parameter(Mandatory = $true)]
+    [string]$SourceCommish,
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+  )
+  $repoFullName = "$RepoOwner/$RepoName"
+  $apiviewEndpoint = "https://apiview.dev/api/pullrequests?pullRequestNumber=$PrNumber&repoName=$repoFullName&commitSHA=$SourceCommish"
```

Comment:
Let's add this url as hidden text in the comment so we can use it for debugging issues with this.